ui/next.config.js

```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
 
 module.exports = withTM({
   experimental: { esmExternals: true },
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: securityHeaders,
+      },
+    ]
+  },
 })
+
+// The declaration below will allow Youtube Videos and Google Fonts
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' *.youtube.com;
```

Comment:
I'm not sure if we need unsafe-eval e unsafe-inline

Reply:
It seems that we need them at least for development, not sure about production.
![Screenshot 2021-09-09 às 14 29 31](https://user-images.githubusercontent.com/566971/132734119-859376e3-a235-41a7-8c30-51c542d13153.png)

I know Next injects styles in Dev and not in production, but about the scripts we'll probably need more allowance.

I'll play a little bit to find a good balance

Reply:
Now we'll only have `unsafe-eval` in development ;)